src/btree/bt_page.c

```diff
@@ -660,10 +660,11 @@ __inmem_row_int(WT_SESSION_IMPL *session, WT_PAGE *page, size_t *sizep)
 
     /*
      * We track if an internal page has backing overflow keys, as overflow keys limit the eviction
-     * we can do during a checkpoint.
+     * we can do during a checkpoint. (This is only for historical tables, reconciliation no longer
```

Comment:
Is a historical table referring to any table before the changes done in the PR?

Reply:
Yes, by historical table, I meant tables created before this change.